src/native/managed/compile-native.proj

```diff
@@ -13,26 +13,11 @@
         <NativeLibsProjectsToBuild Include="$(MSBuildThisFileDirectory)cdacreader/src/cdacreader.csproj" />
     </ItemGroup>
 
-    <!-- Decide if we're going to do the NativeAOT builds -->
-    <PropertyGroup>
-        <!-- disable on Mono, for now -->
-        <SupportsNativeAotComponents Condition="'$(SupportsNativeAotComponents)' == '' and '$(RuntimeFlavor)' == 'Mono'">false</SupportsNativeAotComponents>
```

Comment:
Cc @lambdageek - we might be losing this exclusion, but that doesn't seem harmful on its own.

Reply:
yea it's ok.  I don't think there's anything that depends on `compile-native.proj` when we're building Mono, so this check was probably redundant

Reply:
I think in an earlier iteration of the cdacreader build infrastructure I had compile-native as a separate subset and it was getting pulled in from some mono cross-compiler scenarios.  But now it's only pulled in from coreclr's runtime-prereqs.proj, which shouldn't be triggered by any mono builds